crates/bdk/src/descriptor/error.rs

```diff
@@ -79,6 +84,7 @@ impl fmt::Display for Error {
             Self::Pk(err) => write!(f, "Key-related error: {}", err),
             Self::Miniscript(err) => write!(f, "Miniscript error: {}", err),
             Self::Hex(err) => write!(f, "Hex decoding error: {}", err),
+            Self::NonUniqueInternal(d) => write!(f, "Non unique change descriptor: {}", d),
```

Comment:
Same for all these `write!`s:

```suggestion
            Self::NonUniqueInternal(d) => write!(f, "Non unique change descriptor: {d}"),
```

Reply:
https://github.com/bitcoindevkit/bdk/pull/1390#discussion_r1545444089